spec/tiler_spec.coffee

```diff
@@ -1,92 +1,198 @@
 describe 'Tiler', ->
+  $tiler = null
+
   describe 'initialize', ->
     before ->
-      $('#initialize').tiler()
+      $tiler = $('#initialize')
+      $tiler.tiler
+        activeTile: 1
+        isReversible: true
 
     it 'should not include tiles from nested tiler instances', ->
-      expect($('#initialize > .tiler-tile').data 'tiler-viewport-id').to.equal 'initialize'
-      expect($('#initialize-nested > .tiler-tile').data 'tiler-viewport-id').to.not.equal 'initialize'
+      expect($('.tiler-tile', $tiler).data 'tiler-viewport-id').to.equal 'initialize'
+      expect($('#initialize-nested > .tiler-tile').data 'tiler-viewport-id').to.be.undefined
+
+    it 'should setup enter and exit tiles', ->
+      # tile 1
+      $activeTile = $('#tile-initialize-1', $tiler)
+      expect($activeTile).to.have.class 'tile-initialize-1-animation'
+      expect($activeTile).to.have.class 'active'
```

Comment:
Are all these particular classes and what they mean documented somewhere? This has always been a bit confusing from a developer's perspective.